package.json

```diff
@@ -48,12 +48,21 @@
     "@commitlint/cli": "^17.6.5",
     "@commitlint/config-conventional": "^17.6.5",
     "@swc/core": "^1.3.66",
+    "assert": "^2.0.0",
+    "buffer": "^5.5.0||^6.0.0",
```

Comment:
If it is only needed in the `packages/app` we can leave it there inside `app/package.json`.